README.md

```diff
@@ -208,15 +211,21 @@ iOS13 and higher
 iOS12 and lower
 * NSBluetoothPeripheralUsageDescription
 
-#### How to adjust ProGuard (Android)
+#### How to adjust ProGuard (Android)?
```

Comment:
I would suggest to remove (revert) the question mark in the title.

An external example: https://support.microsoft.com/en-us/windows/how-to-refresh-reset-or-restore-your-pc-51391d9a-eb0a-84a7-69e4-c2c1fbceb8dd